docs/topics/crud-README.adoc

```diff
@@ -27,4 +27,4 @@ endif::[]
 = ${mission} - ${runtime} Booster
 
 
-IMPORTANT: While you can run and interact with this booster on our local host, it requires that you also have a database installed and configured. This booster runs best when deployed on OpenShift with a PostGres database. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
\ No newline at end of file
+IMPORTANT: While you can run and interact with this booster on our local host, it requires that you also have a database installed and configured. This booster runs best when deployed on OpenShift with a PostGreSQL database. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
```

Comment:
OK so historically, PostgreSQL is "Post Ingres", but the correct name is PostgreSQL now.